src/data/history.ts

```diff
@@ -223,36 +223,32 @@ export const fetchDate = (
   hass: HomeAssistant,
```

Comment:
Is `fetchDate` dead code now?

Reply:
I think it may be, but I didn't remove it, it was changed to fetchDateWS through the duration of this PR

Reply:
By a concurrent commit